Dockerfile

```diff
@@ -49,8 +49,7 @@ COPY bff/go.mod bff/go.sum ./
 RUN go mod download
 COPY bff/ ./
 RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.55.2
-COPY bff/solid/ ./solid
-RUN cd ./solid && npm install && npx @biomejs/biome check .
+RUN cd ./solid && npx @biomejs/biome check --config-path=. .
```

Comment:
```suggestion
RUN cd ./bff/solid && npx @biomejs/biome check --config-path=. .
```

Reply:
The directory is right, because 1) it is relative to the bff directory which was copied to "./" 2 lines above it, and 2) because it works when you run `docker build . --target lint`

Reply:
Ah, I see. Thanks for the clarification.